config/kubernetes-sigs/sig-docs/teams.yaml

```diff
@@ -2,7 +2,9 @@ teams:
   reference-docs-admins:
     description: admin access to the reference-docs repo
     members:
-    - jimangel
+    - divya-mohan0209
+    - natalisucks
+    - reylejano
```

Comment:
/approve

Will leave lgtm to a sig docs for these changes.
/assign @divya-mohan0209